examples/github/binding.yaml

```diff
@@ -0,0 +1 @@
+../role-resources/triggerbinding-roles/binding.yaml
```

Comment:
As I mentioned in #578 , I'd prefer adding the binding/template/pipeline/secret in this folder in the same file as it makes it much clearer what is going on (just look at this one file vs looking at 6-7 different files). I know this leads to some duplication but for examples I'm ok with having some repetition in order to help with clarity